<launch>
    <!--Parameters-->
    <arg name="frame_id" default="map" />

    <!--node launch-->
    <node pkg= "decawave" ns="uwb" name="pdoaLocatorFiltered" type="pdoaLocateVICON" output="screen">
        <param name="frame_id" value="$(arg frame_id)" />
        <param name="pdoaTopic" value="/uwb/pdoaLocator/position" />
        <param name="viconTopic" value="/vicon/pdoa_tag/pose" />

    </node>

</launch>
